lib/pbench/cli/server/pbench_create_user.py

```diff
@@ -0,0 +1,94 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-create-user"
+
+
+class PbenchCreateUser:
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        try:
+            config = PbenchServerConfig(self.context.config)
+        except BadConfig:
+            raise
+
+        try:
+            logger = get_pbench_logger(_NAME_, config)
+
+            # We're going to need the Postgres DB to track dataset state, so setup
+            # DB access.
+            Database.init_db(config, logger)
+
+            user = User(
+                username=self.context.username,
+                password=self.context.password,
+                first_name=self.context.first_name,
+                last_name=self.context.last_name,
+                email=self.context.email,
+            )
+            user.add()
+            click.echo(f"User {self.context.username} registered")
+            return 0
```

Comment:
Given that you report all errors via exceptions, you don't need to return any value on success (i.e., you can just omit this line here and let the caller generate the exit status value).